infoschema/perfschema/const.go

```diff
@@ -447,18 +448,21 @@ const tableEventsStatementsSummaryByDigest = "CREATE TABLE if not exists " + tab
 	"MAX_PREWRITE_REGIONS INT(11) UNSIGNED NOT NULL," +
 	"AVG_TXN_RETRY DOUBLE NOT NULL," +
 	"MAX_TXN_RETRY INT(11) UNSIGNED NOT NULL," +
+	"SUM_BACKOFF_TIMES BIGINT(20) UNSIGNED NOT NULL," +
 	"BACKOFF_TYPES VARCHAR(1024) DEFAULT NULL," +
 	"AVG_MEM BIGINT(20) UNSIGNED NOT NULL," +
 	"MAX_MEM BIGINT(20) UNSIGNED NOT NULL," +
 	"AVG_AFFECTED_ROWS DOUBLE UNSIGNED NOT NULL," +
 	"FIRST_SEEN TIMESTAMP(6) NOT NULL," +
 	"LAST_SEEN TIMESTAMP(6) NOT NULL," +
-	"QUERY_SAMPLE_TEXT LONGTEXT DEFAULT NULL);"
+	"QUERY_SAMPLE_TEXT LONGTEXT DEFAULT NULL," +
+	"PREV_SAMPLE_TEXT LONGTEXT DEFAULT NULL);"
 
 // tableEventsStatementsSummaryByDigestHistory contains the column name definitions for table
 // events_statements_summary_by_digest_history.
 const tableEventsStatementsSummaryByDigestHistory = "CREATE TABLE if not exists events_statements_summary_by_digest_history (" +
 	"SUMMARY_BEGIN_TIME TIMESTAMP(6) NOT NULL," +
+	"SUMMARY_END_TIME TIMESTAMP(6) NOT NULL," +
```

Comment:
We can use a constant format string to save the create table SQL if the `digest` and the `history digest` table have the same schema.

Reply:
Good, done.